contracts/RewardsStandaloneSingleToken.sol

```diff
@@ -0,0 +1,214 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.7.6;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import "./interfaces/IBarn.sol";
+import "./interfaces/ISmartYield.sol";
+import "hardhat/console.sol";
+
+contract RewardsStandaloneSingleToken is Ownable {
```

Comment:
Maybe a naive question but why does this contract which looks specific to Smart Yield is added as part of this repository which holds the DAO contracts vs a separate repo?

Reply:
Initially, it was derived from the DAO rewards contract which is the reason it was in the same repo. In the meantime it was extracted to a different repository. I'm gonna close this PR